execute.go

```diff
@@ -70,16 +69,12 @@ func executeWithOutputChan(outputChan chan string, env []string, workingDir stri
 		return nil, fmt.Errorf("error starting process: %v", startErr)
 	}
 
-	waitErr := cmd.Wait()
+	//create a new effectiveconfig instance
+	var con = new(effectiveConfig)
```

Comment:
<pre> var con = new(effectiveConfig)
buildTimeOut := con.BuildTimeout()</pre>

gives me the defaultTimeOutSeconds, so it is not empty. Looking at passing in option.